synapse/lib/stormlib/spooled.py

```diff
@@ -37,6 +37,7 @@ def getObjLocals(self):
     async def _methSet(self, *vals):
         core = self.runt.snap.core
         spool = await s_spooled.Set.anit(dirn=core.dirn, cell=core, size=1000)
+        self.runt.onfini(spool)
```

Comment:
Do we have scoping problems here, with a object being created in one runtime and making it to a parent runtime? Should we instead be finin'ing these on the fini of the root runtime?